sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -131,6 +131,8 @@ class BlobClient(StorageAccountHostsMixin, StorageEncryptionMixin):  # pylint: d
         the exceeded part will be downloaded in chunks (could be parallel). Defaults to 32*1024*1024, or 32MB.
     :keyword int max_chunk_get_size: The maximum chunk size used for downloading a blob. Defaults to 4*1024*1024,
         or 4MB.
+    :keyword str version_id: The version id parameter is an opaque DateTime value that, when present,
```

Comment:
Adding this explicit unresolved feedback so that I remember to add a changelog detailing all these QoL improvements + API changes (not sure if we will merge before next release, so holding off on adding to changelog until it's merging time)